app/models/submission.rb

```diff
@@ -223,6 +223,13 @@ def autograde_feedback_path
     old_autograde_feedback_path
   end
 
+  def create_user_directory_and_return_autograde_feedback_path
+    create_user_handin_directory
+
+    File.join(assessment.handin_directory_path, course_user_datum.email,
```

Comment:
nit: maybe consolidate autograde feedback path into a function or something? (duplicated on 214)

Reply:
I was following what `create_user_directory_and_return_handin_file_path` did, which also duplicates the logic

I'll extract the new paths into helper methods